addons/web/static/tests/views/fields/phone_field_tests.js

```diff
@@ -184,5 +185,10 @@ QUnit.module("Fields", (hooks) => {
             target.querySelector(".o_field_widget[name='foo'] input").placeholder,
             "Placeholder"
         );
+        assert.containsNone(
+            target.querySelector("div:nth-child(2) .o_field_widget[name='foo'] input"),
+            "a",
+            "The readonly field don't contain a link if no value is set"
+        );
```

Comment:
it's weird to have this assertion in this test. I would add another test "unset and readonly PhoneField", for instance